block/meta-iosched.c

```diff
@@ -209,15 +211,16 @@ bool del_queue(struct request_queue *q) {
 
 int whole_stat = 0;
 
-void print_stats(void) {
+void print_stats(unsigned long unused) {
 	int i;
-	printk("Stats! ");
```

Comment:
printk needs KERN_<level> before the message. Better use pr_debug/pr_info/pr_err.